backend/src/main/java/com/onwelo/practice/bts/entity/BankAccount.java

```diff
@@ -20,11 +23,13 @@
 
     private Float moneyBlocked;
 
-    @OneToMany(mappedBy = "destination", targetEntity = Transfer.class, fetch = FetchType.EAGER)
-    private Collection incomingTransfers;
+    @OneToMany(mappedBy = "destination", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @OnDelete(action = OnDeleteAction.CASCADE)
```

Comment:
Lepiej chyba będzie dorzucić opcję cascade w adnotacji `@OneToMany` zamiast dodawać kolejną. Tym bardziej że dzięki temu nie będziemy się sztywno wiązać z implemetnacją jaką jest hibernate.
Co wiecej nie wiem czy chcemy usuwać przelewy. To są raczej wrażliwe dane, więc pewnie nawet jeśli ktoś usunie konto to będziemy chcieli mieć podgląd tego co się kiedyś działo.